fdbservice/CMakeLists.txt

```diff
@@ -3,4 +3,4 @@ add_executable(fdbmonitor ${FDBSERVICE_SRCS})
 
 get_target_property(fdbclient_target_includes fdbclient INCLUDE_DIRECTORIES)
 target_link_libraries(fdbmonitor PUBLIC SimpleOpt)
-target_include_directories(fdbmonitor PUBLIC "${fdbclient_target_includes}")
\ No newline at end of file
+target_include_directories(fdbmonitor PUBLIC "${fdbclient_target_includes}" "${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
Why is this needed?

Reply:
This is from a test fix for windows build, I will remove it.